bedrock/products/templates/products/vpn/resource-center/landing.html

```diff
@@ -0,0 +1,86 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from 'macros-protocol.html' import call_out, split %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('vpn-resource-center')}}
+{% endblock page_css %}
+
+{% block page_title%}{{ ftl('vpn-resource-center-title') }}{% endblock page_title %}
+
+{% block content %}
+  {{ call_out (
+    title=ftl('vpn-resource-center-whats-the-deal'),
+    desc=ftl('vpn-resource-center-discover-how-they'),
+    class='resource-center-page-header resource-center-hero'
+  )}}
+    {# Category component is currently of the scope of this issue, but still stubbed out for use later #}
+
+    {# {% if category_list|length > 1 %}
+    {# <div class="mzp-c-category-list"> #}
+      {# <h2 class="mzp-c-category-list-title"> #}
+        {# <a href="{{ url('products.vpn.resource-center.landing') }}">View All &#62</a>
+      </h2>
+      <ul class="mzp-c-category-list-list">
+        {% for category in category_list  %}
+        <li class="mzp-c-category-list-item">
+          {# TODO? if selected_category == category.name, add selected state #}
+          {# <a href="{{category.url}}">{{category.name}}</a> #}
+        {# </li> #}
+        {# {% endfor %} #}
+      {# </ul> #}
+    {# </div> #}
+    {# {% endif %} #}
+
+  <div class="mzp-l-content mzp-l-card-third">
+    {% for article in first_article_group %}
+      {% include "products/vpn/resource-center/includes/vpn-article-card.html" %}
+    {% endfor %}
+  </div>
+  {% call split (
+    image_url="img/products/vpn/resource-center/resource_center_split_1.svg",
+    block_class='resource-center-split mzp-t-split-nospace mzp-t-content-lg mzp-l-split-center-on-sm-md',
+    media_class='resource-center-split-media',
+  ) %}
+    <h1 class="resource-center-wordmark">{{ ftl('vpn-resource-center-mozilla-vpn') }}</h1>
+    <h2 class="mzp-c-call-out-title">{{ ftl('vpn-resource-center-start-protecting') }}</h2>
+    <a class='mzp-c-button mzp-t-product resource-center-cta'>{{ ftl('vpn-resource-center-on-mozilla-vpn', amount=vpn_saving(plan='12-month', country_code=country_code, lang=LANG)) }}</a>
```

Comment:
Looks like this one is still a non-functioning link? suggest changing it to `vpn_product_referral_link`.